packages/web-api/test/types/methods/slacklists.test-d.ts

```diff
@@ -0,0 +1,161 @@
+expectError(web.slackLists.download.get({ list_id: 'L1234567890' })); // missing job_id
+
+// -- happy path
+expectAssignable<Parameters<typeof web.slackLists.download.get>>([
+  {
+    list_id: 'L1234567890',
+    job_id: 'J1234567890',
+  },
+]);
+
+// slackLists.download.start
+// -- sad path
+expectError(web.slackLists.download.start()); // lacking argument
+expectError(web.slackLists.download.start({})); // missing list_id
+
+// -- happy path
+expectAssignable<Parameters<typeof web.slackLists.download.start>>([
+  {
+    list_id: 'L1234567890',
+  },
+]);
+
+// slackLists.items.create
+// -- sad path
+expectError(web.slackLists.items.create()); // lacking argument
+expectError(web.slackLists.items.create({})); // missing list_id
+
+// -- happy path
+expectAssignable<Parameters<typeof web.slackLists.items.create>>([
+  {
+    list_id: 'L1234567890',
+  },
+]);
+
+// slackLists.items.delete
+// -- sad path
+expectError(web.slackLists.items.delete()); // lacking argument
+expectError(web.slackLists.items.delete({})); // missing list_id and id
+expectError(web.slackLists.items.delete({ list_id: 'L1234567890' })); // missing id
+
+// -- happy path
+expectAssignable<Parameters<typeof web.slackLists.items.delete>>([
+  {
+    list_id: 'L1234567890',
+    id: 'I1234567890',
+  },
+]);
+
+// slackLists.items.deleteMultiple
+// -- sad path
+expectError(web.slackLists.items.deleteMultiple()); // lacking argument
+expectError(web.slackLists.items.deleteMultiple({})); // missing list_id and ids
+expectError(web.slackLists.items.deleteMultiple({ list_id: 'L1234567890' })); // missing ids
+
+// -- happy path
+expectAssignable<Parameters<typeof web.slackLists.items.deleteMultiple>>([
+  {
+    list_id: 'L1234567890',
+    ids: ['I1234567890', 'I0987654321'],
+  },
+]);
+
+// slackLists.items.info
+// -- sad path
+expectError(web.slackLists.items.info()); // lacking argument
+expectError(web.slackLists.items.info({})); // missing list_id and id
+expectError(web.slackLists.items.info({ list_id: 'L1234567890' })); // missing id
+
+// -- happy path
+expectAssignable<Parameters<typeof web.slackLists.items.info>>([
+  {
+    list_id: 'L1234567890',
+    id: 'I1234567890',
+  },
+]);
+
+// slackLists.items.list
+// -- sad path
+expectError(web.slackLists.items.list()); // lacking argument
+expectError(web.slackLists.items.list({})); // missing list_id
+
+// -- happy path
+expectAssignable<Parameters<typeof web.slackLists.items.list>>([
+  {
+    list_id: 'L1234567890',
+  },
+]);
+
+// slackLists.items.update
+// -- sad path
+expectError(web.slackLists.items.update()); // lacking argument
+expectError(web.slackLists.items.update({})); // missing list_id and cells
+expectError(web.slackLists.items.update({ list_id: 'L1234567890' })); // missing cells
+
+// -- happy path
+expectAssignable<Parameters<typeof web.slackLists.items.update>>([
+  {
+    list_id: 'L1234567890',
+    cells: [{ id: 'C1234567890', value: 'test' }],
```

Comment:
🚧 question: Is `id` a valid value for a "cell" here? I'm not finding this in documentation at the moment...

🔗 https://docs.slack.dev/reference/methods/slackLists.items.update#sample-requests-data